ksqldb-streams/src/test/java/io/confluent/ksql/execution/streams/SourceBuilderTest.java

```diff
@@ -645,6 +665,7 @@ private void givenMultiColumnSourceTable() {
   }
 
   private <K> void givenConsumed(final Consumed<K, GenericRow> consumed, final Serde<K> keySerde) {
+    when(consumedWindowed.withValueSerde(any())).thenReturn(consumedWindowed);
```

Comment:
I don't understand -- how come we're passing in `consumed` as a parameter but mocking behavior on the class member `consumedWindowed` instead?